pyriemann/transferlearning.py

```diff
@@ -0,0 +1,144 @@
+
+import numpy as np
+from sklearn.model_selection import KFold
+from sklearn.base import BaseEstimator, TransformerMixin, ClassifierMixin
+from pyriemann.utils.mean import mean_riemann
+from pyriemann.utils.base import invsqrtm
+
+
+class TLSplitter():
+    def __init__(self, n_splits=5):
+        self.n_splits = n_splits
+
+    def split(self, X, y, meta):
+
+        # index of all source data points
+        idx_source = np.where(~meta['target'])[0]
+
+        # index of training-split for the target data points
+        kf_target = KFold(n_splits=self.n_splits).split(X[meta['target']])
+        for train_idx_target, test_idx_target in kf_target:
+            train_idx_target = list(
+                meta[meta['target']].index[train_idx_target])
+            train_idx = np.concatenate([idx_source, train_idx_target])
+            test_idx_target = list(meta[meta['target']].index[test_idx_target])
+            test_idx = test_idx_target
+            yield train_idx, test_idx
+
+    def get_n_splits(self, X, y, meta):
+        return self.n_splits
+
+
+class DCT(BaseEstimator, TransformerMixin):
+    '''
+    No transformation of the data points between the domains.
+    This is what we call the direct (DCT) method.
```

Comment:
Class name `DCT` reminds us of Discrete Cosinus Transform...
What about another name? `DummyAdapter`?

Reply:
I changed the name to `TLDummy` in transferlearning_yenc